tasks/utilities/user_role_sync/user_role_sync.rb

```diff
@@ -82,10 +82,17 @@ def run(options)
         @remove_users = @options[:remove_users]
         @role_exclusions = @options[:role_exclusions] # || ""
 
+        api_client = Kenna::Api::Client.new(@api_token, @api_host)
+        @version = api_client.version
+
         # Variables we'll need later
         @role_post_url = "https://#{@api_host}/roles"
         @user_post_url = "https://#{@api_host}/users"
-        @headers = { "content-type" => "application/json", "X-Risk-Token" => @api_token, "User-Agent" => "Kenna Toolkit - user_role_sync" }
+        @headers = {
+          "content-type" => "application/json",
+          "X-Risk-Token" => @api_token,
+          "User-Agent" => "Toolkit.user_role_sync/1.0.#{@version} (Kenna Security)"
```

Comment:
I agree it would be good to get the header-building in one spot.  I guess it would need to accept the role-name as a parameter.

Also, It would be good if Kenna::Api::Client knew of the whole MAJOR.MINOR.REVISION of the version string.

Reply:
I will put the header in one place, but the version I will change the version when I get the GitHub hook working.